pkg/relayer/proxy/websockets/bridge.go

```diff
@@ -0,0 +1,443 @@
+package websockets
+
+import (
+	"context"
+	"net/http"
+	"sync"
+
+	"github.com/gorilla/websocket"
+
+	"github.com/pokt-network/poktroll/pkg/client"
+	"github.com/pokt-network/poktroll/pkg/observable/channel"
+	"github.com/pokt-network/poktroll/pkg/polylog"
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	"github.com/pokt-network/poktroll/pkg/relayer/config"
+	"github.com/pokt-network/poktroll/x/service/types"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+)
+
```

Comment:
Can you please create a new page in the docs and just copy-paste this in there with a TODO explaining it's a disclaimer as of PR #1073?

I just want a new page in place to link people to that we'll iterate on. This is a great explanation.

Reply:
`docusaurus/docs/protocol/primitives/async_service_mining.md`